plugins/outputs/zabbix/autoregister_test.go

```diff
@@ -0,0 +1,118 @@
+package zabbix
+
+import (
+	"testing"
+	"time"
+
+	"github.com/datadope-io/go-zabbix/v2"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+// TestZabbixAutoregisterDisabledAdd tests that Add does not store information if autoregister is disabled.
+func TestZabbixAutoregisterDisabledAdd(t *testing.T) {
+	z := Zabbix{
+		autoregisterLastSend: make(map[string]time.Time),
+	}
+
+	z.autoregisterAdd("hostname")
+	assert.Equal(t, len(z.autoregisterLastSend), 0)
```

Comment:
Can you please use the `require` equivalents throughout your tests. In most cases you can simple search-and-replace.

Reply:
Fixed in 16be01e52